src/backend/distributed/commands/database.c

```diff
@@ -302,7 +302,7 @@ PreprocessCreateDatabaseStmt(Node *node, const char *queryString,
 		return NIL;
 	}
 
-	EnsureCoordinator();
+	EnsurePropagationToCoordinator();
```

Comment:
Are you sure that EnsurePropagationToCoordinator covered?

Reply:
Indeed I can see the error thrown in the test file but maybe codecov is complaining about some branch coverage problems (as we've discussed yesterday) that I'm not aware of.

Reply:
As we can see in `create_drop_database_propagation.out`:

```sql
\c - - - :worker_1_port
SET citus.enable_create_database_propagation TO ON;
-- fails because coordinator is not added into metadata
DROP DATABASE another_db_created_from_worker;
ERROR:  coordinator is not added to the metadata
HINT:  Use SELECT citus_set_coordinator_host('<hostname>') to configure the coordinator hostname
```